packages/aws-cdk-lib/aws-stepfunctions-tasks/lib/eventbridge/put-events.ts

```diff
@@ -132,23 +132,23 @@ export class EventBridgePutEvents extends sfn.TaskStateBase {
   }
 
   private renderEntries(): Object[] {
+    // we should have validated all entries in validateEntries()
```

Comment:
Is it guaranteed that `validateEntries()` will always be called before `renderEntires()`? I just wonder if there's any cases where `renderEntries()` is called on it's own and removing the check from this method would mean the check isn't run at all.